src/routes/view-content/data-fetchers.ts

```diff
@@ -9,15 +9,24 @@ import { isOnline } from '$lib/stores/is-online.store';
 import { fetchAllBibles, bookDataForBibleTab } from '$lib/utils/data-handlers/bible';
 import { range } from '$lib/utils/array';
 import { type ResourceContentInfo, ParentResourceType } from '$lib/types/resource';
-import { resourceContentApiFullUrl } from '$lib/utils/data-handlers/resources/resource';
+import {
+    resourceContentApiFullUrl,
+    type ResourceContentInfoWithFrontendData,
+} from '$lib/utils/data-handlers/resources/resource';
 import { preferredBibleIds } from '$lib/stores/preferred-bibles.store';
 import { log } from '$lib/logger';
 import { settings } from '$lib/stores/settings.store';
 import { SettingShortNameEnum, type Setting } from '$lib/types/settings';
 import { resourceContentsForBibleSection } from '$lib/utils/data-handlers/resources/resource';
 import type { BibleSection } from '$lib/types/bible';
 
-export type PassagePageTab = 'bible' | 'guide' | 'mainMenu' | 'libraryMenu' | 'resources';
+export enum PassagePageTabEnum {
+    bible = 'bible',
+    guide = 'guide',
+    mainMenu = 'mainMenu',
+    libraryMenu = 'libraryMenu',
+    resources = 'resources',
+}
```

Comment:
Switched this to an enum to make it clearer what some usages were doing.

Reply:
Should the properties still be Pascal case since it's an enum?

Reply:
Yeah I can do that